tox.ini

```diff
@@ -14,6 +14,8 @@ setenv =
     TOXINIDIR = {toxinidir}
 passenv =
     HOME
+deps:
+    setuptools_scm<6
```

Comment:
This is only required while we don't fix https://github.com/pyscaffold/pyscaffold/issues/428

Reply:
Is this still required now after version 4.0.1?

Reply:
No, sorry, I forgot to remove... working on that